scripts/libmetal.mk

```diff
@@ -43,6 +43,10 @@ $(BSP_DIR)/metal.scratchpad.lds: $(BSP_DIR)/design.dts $(LDSCRIPT_GENERATOR)
 	$(MAKE) -f scripts/virtualenv.mk virtualenv
 	. $(FREEDOM_E_SDK_VENV_PATH)/bin/activate && $(LDSCRIPT_GENERATOR) -d $< -o $@ --scratchpad
 
+$(BSP_DIR)/metal.freertos.lds: $(BSP_DIR)/design.dts $(LDSCRIPT_GENERATOR)
+	$(MAKE) -f scripts/virtualenv.mk virtualenv
+	. venv/bin/activate && $(LDSCRIPT_GENERATOR) -d $< -o $@ --freertos
```

Comment:
should be `. $(FREEDOM_E_SDK_VENV_PATH)/bin/activate`

Reply:
Done